windows-installer/setup.nsi

```diff
@@ -185,13 +189,14 @@ Section "Install"
 
   ; Create a directory to store position files.
   CreateDirectory ${MAIN_INSTDIR}\pos
+  ; Create a directory for custom logs.
+  CreateDirectory ${CUSTOM_LOGS_DIR}
```

Comment:
Let's instead create the `"$INSTDIR\config.d"` directory... You're welcome to parameterize it, but I'm fairly certain it'll never change. Since it's relative to the location of the main config file, maybe parameterize that directory instead, e.g.:
```
; Absolute location of the directory containing the fluentd configuration.
 !define FLUENTD_CONFIG_DIRECTORY "$INSTDIR"

; Absolute location of the fluentd config file.
 !define FLUENTD_CONFIG_LOCATION "$FLUENTD_CONFIG_DIRECTORY\fluent.conf"

; Absolute location of the fluentd config extensions.
 !define FLUENTD_CONFIG_EXTENSIONS "$FLUENTD_CONFIG_DIRECTORY\config.d"
```
...
```
  ; Create a directory for config extensions.
  CreateDirectory ${FLUENTD_CONFIG_EXTENSIONS}
```